beaker-vue/src/components/sidemenu/SideMenu.vue

```diff
@@ -458,7 +458,7 @@ defineExpose({
     position: absolute;
     right: -6px;
     width: 14px;
-    z-index: 100;
+    z-index: 31;
```

Comment:
So that the center/main area has higher z-index without having to explode to above-100 for more priority values. Then the center notebook area has higher z-index, and the codemirror tooltip renders on the right location (not under the side panes).